aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/UnifiedPushConfig.java

```diff
@@ -245,7 +245,7 @@ public UnifiedPushConfig setAlias(String alias) {
      * 
      */
     public UnifiedPushConfig setCategories(List<String> categories) {
-        this.categories = new ArrayList<String>(categories);
+        this.categories = categories;
```

Comment:
@secondsun  Yes, try use `setCategories` with [Hello World Project](https://github.com/jboss-mobile/unified-push-helloworld/tree/master/android) and check the categories in UPS